packages/authenticator/amplify_authenticator/lib/amplify_authenticator.dart

```diff
@@ -32,6 +32,7 @@ import 'package:amplify_authenticator/src/utils/dial_code.dart';
 import 'package:amplify_authenticator/src/utils/dial_code_options.dart';
 import 'package:amplify_authenticator/src/widgets/authenticator_banner.dart';
 import 'package:amplify_authenticator/src/widgets/form.dart';
+import 'package:amplify_core/amplify_core.dart' as core;
```

Comment:
The import of amplify_core is needed since AmplifyOutputs is located here:
packages/amplify_core/lib/src/config/amplify_outputs/amplify_outputs.dart

amplify_core and amplify_flutter both have a global variable of Amplify, so I aliased one of them to identify which one is correct. Looking at it again I can simply remove the reference to amplify_flutter since it's Amplify is mainly a wrapper for the amplify_core's Amplify.

Reply:
I also think you can remove the import